.github/workflows/pre_merge.yml

```diff
@@ -25,8 +25,14 @@ jobs:
           export ANOMALIB_DATASET_PATH=/media/data1/datasets/
           export CUDA_VISIBLE_DEVICES=3
           tox -e pre_merge
-      - name: Upload coverage result
-        uses: actions/upload-artifact@v2
-        with:
-          name: coverage
-          path: .tox/coverage.xml
+      - name: Upload coverage report
+        run: |
+
```

Comment:
It might be an idea to explain this `if`-block with a comment for future reference.